source/common/upstream/outlier_detection_impl.cc

```diff
@@ -37,14 +37,10 @@ DetectorHostMonitorImpl::DetectorHostMonitorImpl(std::shared_ptr<DetectorImpl> d
                                                  HostSharedPtr host)
     : detector_(detector), host_(host) {
   // add Success Rate monitors
-  success_rate_monitors_
-      [envoy::data::cluster::v2alpha::OutlierEjectionType::SUCCESS_RATE_EXTERNAL_ORIGIN] =
-          std::make_unique<SuccessRateMonitor>(
-              envoy::data::cluster::v2alpha::OutlierEjectionType::SUCCESS_RATE_EXTERNAL_ORIGIN);
-  success_rate_monitors_
-      [envoy::data::cluster::v2alpha::OutlierEjectionType::SUCCESS_RATE_LOCAL_ORIGIN] =
-          std::make_unique<SuccessRateMonitor>(
-              envoy::data::cluster::v2alpha::OutlierEjectionType::SUCCESS_RATE_LOCAL_ORIGIN);
+  externalOriginSRMonitor_ = std::make_unique<SuccessRateMonitor>(
```

Comment:
nit: snake case, same below, also this can be initialized in the initializer list above.

Reply:
Done. Moved initialization to the list.